modules/income_limits/spec/requests/v1/limits_by_zip_code_spec.rb

```diff
@@ -1,13 +1,154 @@
 # frozen_string_literal: true
 
+# rubocop:disable Layout/LineLength
+
 require 'rails_helper'
 
-RSpec.describe 'Income limits endpoints', type: :request do
-  describe '#get limitsByZipCode/:zip/:year/:dependents' do
-    it 'returns sample data' do
-      get '/income_limits/v1/limitsByZipCode/12345/2022/2'
-      expect(response).to have_http_status(:ok)
-      JSON.parse(response.body)
+RSpec.describe 'IncomeLimits::V1::IncomeLimitsController', type: :request do
+  describe 'GET #index' do
+    def parse_response(response)
+      JSON.parse(response.body)['data']
+    end
+
+    context 'with valid parameters' do
+      let(:zip) { '15222' }
```

Comment:
Can you add another test that has a valid zip code that starts with`0`?

Reply:
there is now a test for 0 leading zip codes